commands/helpcenter.js

```diff
@@ -0,0 +1,44 @@
+const guildConfig = require('../util/guildConfig.js');
+const util = require('../lib/util.js');
+const axios = require('axios');
+
+exports.command = async (message, args, database, bot) => {
+    //Permission check
+    if (!message.member.hasPermission('MANAGE_GUILD')) {
+        await message.channel.send('You need the "Manage Server" permission to use this command.');
+        return;
+    }
+
+    let subdomain = /zendesk.com/.test(args[0]) ? args[0].split(/.zendesk.com/)[0] : args[0];
```

Comment:
Check if the string actually **ends** with **.**zendesk**.**com. Remove invalid characters from the subdomain (Zendesk subdomains can only include alphanumeric characters).

```suggestion
    let subdomain = args[0].replace(/\.zendesk\.com$/i, '').replace(/[^a-zA-Z\d]/g, '');
```